src/components/organisms/SettingsDrawer/SettingsDrawer.tsx

```diff
@@ -93,6 +99,33 @@ const SettingsDrawer = () => {
     }
   };
 
+  const checkUpdateAvailability = () => {
+    console.log('checkUpdateAvailability');
```

Comment:
do we want to keep all these log statements?

Reply:
No, before I change draft PR to a real one I will delete all logs.